addons/website_crm_partner_assign/controllers/main.py

```diff
@@ -133,6 +133,8 @@ def portal_my_opportunities(self, page=1, date_begin=None, date_end=None, sortby
         if date_begin and date_end:
             domain += [('create_date', '>', date_begin), ('create_date', '<=', date_end)]
         # pager
+        leads_sudo = CrmLead.sudo().search(domain)
+        domain = [('id', 'in', leads_sudo.ids)]
         opp_count = CrmLead.search_count(domain)
```

Comment:
Would just add a comment explaining like "bypass activities access rights for search but still apply access rules"